doc/source/getting_started/installing.rst

```diff
@@ -0,0 +1,397 @@
+.. _installing_odl:
+
+##############
+Installing ODL
+##############
+
+Introduction
+============
+
+Installing ODL is intended to be straightforward, and this guide is meant for new users.
+For a minimal installation you should perform the following steps:
+
+1. Install a Python interpreter
+2. Install ODL and its dependencies
+3. (optional) Run the tests
+
+Consider using Anaconda
+-----------------------
+We currently recomment to use `Anaconda`_ on all platforms since it offers the best out-of-the-box installation and run-time experience.
+Anaconda also has other benefits, for example the possibility to work in completely isolated Python environments with own installed packages, thereby avoiding conflicts with system-wide installed packages.
+Furthermore, Anaconda cooperates with ``pip`` (see below), i.e. packages can be installed with both Anaconda's internal mechanism and ``pip`` without conflicts.
+
+Alternatively, packages can be installed with `pip`_ in a user's location, which should also avoid conflicts.
+We will provide instructions for this alternative.
+
+Another possibility is to use `virtualenv`_, which can be seen as a predecessor to Anaconda.
+Following the ``pip`` installation instructions in a ``virtualenv`` without the ``--user`` option should work, but we do not provide explicit instructions for this variant.
+
+Which Python version to use?
+----------------------------
+Any modern Python distribution supporting `NumPy`_ and `SciPy`_ should work for the core library, but some extensions require CPython (the standard Python distribution).
+
+ODL fully supports both Python 2 and Python 3.
+If you choose to use your system Python interpreter (the "pip install as user" variant), it may be a good idea to stick with the default one, i.e. the one invoked by ``python``.
+Otherwise, we recommend using Python 3, since Python 2 support will be discontinued in 2020.
+
+Development enviroment
+----------------------
+Since ODL is object-oriented, using an Integrated Development Environment (IDE) is recommended, but not required.
+The most popular ones are `Spyder`_ which works on all major platforms and can be installed through both ``conda`` and ``pip``, and `PyCharm`_ which can be integrated with any text editor of your choice, such as Emacs or Vim.
+
+
+Alternative 1: Installing a release using ``conda`` (recommended for users)
+===========================================================================
+
+TL;DR
+-----
+- Download and install `Miniconda`_
```

Comment:
I think it's just waste of time and bandwidth. When installing ODL it will update the relevant packages anyway, so you can as well go with the small download.